sim/rogue/runes.go

```diff
@@ -96,7 +96,7 @@ func (rogue *Rogue) applyFocusedAttacks() {
 				isFoKOH = true
 			}
 
-			if !spell.ProcMask.Matches(core.ProcMaskMeleeOrRanged|core.ProcMaskTriggerInstant) || !result.DidCrit() || isFoKOH {
+			if !spell.ProcMask.Matches(core.ProcMaskMeleeOrRanged|core.ProcMaskMeleeDamageProc) || !result.DidCrit() || isFoKOH {
```

Comment:
Tested locally and focused attacks with blazefury seems to still be working correctly so that's awesome 🥂

Reply:
Nice!